hack/build-dev.sh

```diff
@@ -1,79 +0,0 @@
-#!/bin/bash
-
-# Copyright 2019 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-#
-# TODO(akutz) This script can probably be removed once v1alpha2 is released.
-#
-
-set -o errexit
-set -o nounset
-set -o pipefail
-
-# Change directories to the parent directory of the one in which this
```

Comment:
AFAICT all of this should be taken care of  in the Makefile and tilt should be the preferred dev environment.